python/nano/src/bigdl/nano/deps/neural_compressor/inc_api_2.py

```diff
@@ -170,7 +172,7 @@ def new_collate_fn(batch):
 
     q_model = quantization.fit(
         model=model,
-        conf=q_conf,
+        conf=config,
```

Comment:
`config` may contain no `framework`, we should use `q_conf`

Reply:
Done